VirtoCommerce.Storefront.Model/Security/SecurityConstants.cs

```diff
@@ -29,10 +29,23 @@ public static class Roles
             {
                 Id = "org-employee",
                 Name = "Organization employee",
-                Permissions = new string[] { Permissions.CanSeeOrganizationDetail } 
+                Permissions = new string[] { Permissions.CanSeeOrganizationDetail }
             };
 
-            public static readonly IEnumerable<Role> AllRoles = new[] { OrganizationMaintainer, OrganizationEmployee };
+            public static readonly Role ProcurementPortalAdmin = new Role
```

Comment:
Please use the neutral or existing role names `ProcurementPortalAdmin` -> OrganizationMaintainer, `ProcurementPortalAgent` -> OrganizationEmployee